gcc/rust/parse/rust-parse.h

```diff
@@ -622,6 +622,21 @@ template <typename ManagedTokenSource> class Parser
   ManagedTokenSource lexer;
   // The error list.
   std::vector<Error> error_table;
+
+  class InlineModuleStackScope
+  {
+  private:
+    Parser &parser;
+
+  public:
+    InlineModuleStackScope (Parser &parser, std::string name) : parser (parser)
+    {
+      parser.inline_module_stack.emplace_back (std::move (name));
+    }
+    ~InlineModuleStackScope () { parser.inline_module_stack.pop_back (); }
+  };
+  // The names of inline modules while parsing.
+  std::vector<std::string> inline_module_stack;
```

Comment:
can we make a private field?

Reply:
This is already a private field.

Reply:
Ah ok sorry, I was misreading this didn't know the '};', can we keep those private fields together and the private class below? It makes it easier to read for me.